todoman.conf

```diff
@@ -0,0 +1,7 @@
+[main]
```

Comment:
This file, along with the two eggs listed above should not have been committed.

Personally, I used `git add -p` to manually add chunks and make sure I'm not adding anything else accidentally.